hpc/LoadBalancer.hpp

```diff
@@ -116,14 +109,14 @@ std::string submitHQJob()
 
         ++i;
         std::cout << "Waiting for job " << job_id << " to start." << std::endl;
-    } while (waitForHQJobState(job_id, "RUNNING") == false && i < 3 && waitForFile("./urls/url-" + job_id + ".txt", 10) == false);
+    } while (waitForHQJobState(job_id, "RUNNING") == false && i < 3 && waitForFile("./urls/url-" + job_id + ".txt") == false);
```

Comment:
Would be a bit cleaner without this loop. I think if job submission fails, there's something seriously wrong. If there is a case where that really helps though, I'm fine with keeping it.

Reply:
I think we still need to check the job state and the URL file, but we don't need to try multiple times. That's because we need to make sure the server inside the jobs is set up properly, and we can send requests right after this function. Without these waiting, we might miss some requests.